changelog/23621.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+**Vault Proxy Static Secret Caching**: Adds support for caching KVv1 and KVv2 secrets to Vault Proxy.
```

Comment:
Clarify:
"Adds support for static secret (KVv1 and KVv2) caching to Vault Proxy"

Reply:
I like this more, thanks! Updated.